uber/site_sections/groups.py

```diff
@@ -37,7 +37,8 @@ def form(self, session, new_dealer='', first_name='', last_name='', email='', me
             message = check(group)
             if not message:
                 session.add(group)
-                message = session.assign_badges(group, params['badges'], params['badge_type'])
+                ribbon_to_use = c.NO_RIBBON if 'ribbon' not in params else params['ribbon']
```

Comment:
it's because I'm hiding the ribbon dropdown in the 'add dealer' screen, so 'ribbon' doesn't get passed into params there.